metadata.rb

```diff
@@ -5,6 +5,7 @@
 version           "0.0.2"
 
 depends "rbenv"
+depends "postfix"
```

Comment:
Shouldn't it be an optional dependency? I don't know the trends of mail servers today, but I'm sure there will be someone who don't want to use it. So you can add it in the `recommends` setting instead of `depends`.